src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -70,9 +70,35 @@ public ReplicatorTest()
             ReopenDB();
             Database.Delete("otherdb", Directory);
             _otherDB = OpenDB("otherdb");
+            Database.SetLogLevel(LogDomain.Replicator, LogLevel.Verbose);
         }
 
-        #if COUCHBASE_ENTERPRISE
+        [Fact]
+        public async Task TestCloseServerWhenReplicatorExistsAsync()
```

Comment:
The point of the test is to check that the replicator stops when it cannot connect to a non-existent endpoint so I'd change the name to reflect that